### PR TITLE
chore: bug report template

### PR DESCRIPTION
**Pull request by andersfylling**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: Bug Report
+description: File a bug report
```
Comment (bkuzma) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: Are you sure this template is valid? According to the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax):

> All issue form configuration files must begin with `name`, `description`, and `body` key-value pairs.

Comment (andersfylling) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: Good catch! I'm leaving it as an empty array

Comment (bkuzma) [reply .github/ISSUE_TEMPLATE/bug_report.md:2]: I opened another PR here with templates auto-generated by Github: https://github.com/cognitedata/cognite-sdk-js/pull/945

**Comment by BugGambit** [comment]
@andersfylling  I guess we can close this due to #945 ?

**Comment by andersfylling** [comment]
Yes

**Pull request closed without merging**